modules/storages/app/components/storages/admin/forms/automatically_managed_project_folders_form_component.rb

```diff
@@ -57,14 +57,6 @@ def cancel_button_options
 
     private
 
-    def submit_button_label
-      if storage.automatic_management_enabled?
-        I18n.t("storages.buttons.done_complete_setup")
-      else
-        I18n.t("storages.buttons.complete_without_setup")
```

Comment:
:question: I am surprised to see that this translation is not removed. Is it still being used somewhere?